tests/testconfig.py

```diff
@@ -1,4 +1,4 @@
-# Github actions template config
+# Github actions template config.
```

Comment:
A lot of those are just because, for example 4 lines below, the dot is used.

Reply:
Btw. all these values not intended to be changed at runtime, i.e. python "constants", should be capitalized.